main.py

```diff
@@ -293,10 +308,10 @@ def sell():
                 for coin in list(order):
 
                     # store some necesarry trade info for a sell
-                    stored_price = float(coin['price'])
+                    stored_price = float(coin['fills'][0]['price'])
                     coin_tp = coin['tp']
                     coin_sl = coin['sl']
-                    volume = coin['executedQty']
+                    volume = round(float(coin['executedQty']) - float(coin['fills'][0]['commission']),2)
```

Comment:
executedQty in the response is the amount of coin purchased, already has the commission removed